sample-compose/build.gradle.kts

```diff
@@ -67,45 +67,26 @@ detekt {
 koverMerged {
     enable()
 
-    val generatedFiles = setOf(
-        "*.R.class",
-        "*.R\$*.class",
-        "*.*\$ViewBinder*.*",
-        "*.*\$InjectAdapter*.*",
-        "*.*Injector*.*",
+    val excludedFiles = listOf(
         "*.BuildConfig.*",
         "*.BuildConfig",
-        "*.Manifest*.*",
-        "*.*_ViewBinding*.*",
-        "*.*Adapter*.*",
-        "*.*Test*.*",
         // Enum
         "*.*\$Creator*",
-        // Nav Component
-        "*.*_Factory*",
-        "*.*FragmentArgs*",
-        "*.*FragmentDirections*",
-        "*.FragmentNavArgsLazy.kt",
-        "*.*Fragment*navArgs*",
-        "*.*ModuleDeps*.*",
-        "*.*NavGraphDirections*",
+        // DI
+        "*.di.*",
         // Hilt
         "*.*_ComponentTreeDeps*",
         "*.*_HiltComponents*",
         "*.*_HiltModules*",
         "*.*_MembersInjector*",
-        "*.Hilt_*"
-    )
-
-    val excludedPackages = setOf(
-        "com.bumptech.glide.*",
-        "dagger.hilt.internal.*",
-        "hilt_aggregated_deps.*",
```

Comment:
@luongvo We should probably include these two lines in `excludedFiles`, I can't see the benefit they bring to the coverage report 🙏🏻 

![Screenshot 2566-05-25 at 10 15 12](https://github.com/nimblehq/android-templates/assets/53168251/de1fe339-f01c-4d45-b257-99d9d31fe07e)

The rest LGTM 👍🏻

Reply:
@ryan-conway Nice spot, added back in https://github.com/nimblehq/android-templates/pull/471/commits/1e3c6a92271cd5ce76ff827a46b368ff77d11142